events/migrations/0001_initial.py

```diff
@@ -0,0 +1,30 @@
+# Generated by Django 4.0.3 on 2022-03-22 17:51
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    initial = True
+
+    dependencies = [
+    ]
+
```

Comment:
Please add some test data, as you can see in Yael's PR linked below
https://github.com/redhat-beyond/lets-meet/blob/4a43276a865c15a024686ff3fec53217c1563b5e/users/migrations/0003_user_test_data.py

In addition, another example can be seen at 
https://drive.google.com/file/d/1BM_P51iCnlRs9OVxTx2woxjfO-FoukRr/view
pages 44 - 45

Reply:
done, added some test data